src/components/editor/components/RecipientInput.scss

```diff
@@ -1,82 +1,119 @@
 @import "../../../res/styles/_required";
 
-.recipients-input tags-input .tags .tag-item {
-  font-family: $font-family-base;
-  font-weight: 500;
-  background-color: theme-color-level("primary", $alert-bg-level);
-  border-color: theme-color-level("primary", $alert-border-level);
-  transition: background-color 0.3s, border-color 0.3s;
-}
-
-@mixin tag-variant($background, $border, $color) {
-  color: $color;
-  @include gradient-bg($background);
-  border-color: $border;
-}
-
-.recipients-input tags-input .tags .remove-button {
-  color: inherit;
-}
-
-@each $color, $value in $theme-colors {
-  .recipients-input tags-input .tags .tag-#{$color} {
-    @include alert-variant(theme-color-level($color, $alert-bg-level), theme-color-level($color, $alert-border-level), theme-color($color));
+@import "/node_modules/bootstrap/scss/badge";
```

Comment:
We currently use src/main.scss in all our html files and it contains all imports listed here. Is it required to import them again for the SCSS build process?

Reply:
I investigated and it turned out to be a difference
2.46 MB for the not modified VS 2.12 MB for the version without bootstrap imports